src/components/PeriodSelector/styles/InputLabel.style.js

```diff
@@ -0,0 +1,8 @@
+import css from 'styled-jsx/css'
+
+export default css`
+    .input-label {
+        color: #595959;
```

Comment:
`ui-core` colors?

Reply:
It is not a ui-core color, but maybe it could be.  (The ui-core colors have a blue-ish tone, while this one has a red-ish tone).

Reply:
I'll address it when PeriodSelector is refactored.